src/controllers/illness/illness.end.date.controller.ts

```diff
@@ -148,6 +163,16 @@ export const processForm = [extractFullDate, ...validators,
   }
 }];
 
+const updateDateErrorMessage = (errorMessage: string, dataToAppend: string, isFirstError: boolean): string => {
```

Comment:
This method is duplicated, should move it to a new location that can be accessed by both controllers. Possibly a new file if you cant find a suitable one.